src/tools/ceph_kvstore_tool.cc

```diff
@@ -39,8 +39,9 @@ class StoreTool
 #ifdef HAVE_LIBAIO
   struct Deleter {
     BlueStore *bluestore;
-    Deleter(BlueStore *store = nullptr)
-    : bluestore(store)
+    Deleter() {}
```

Comment:
I might want to assign bluestore with nullptr

Reply:
@ifed01 fixed and repushed.